website/content/docs/agent/options.mdx

```diff
@@ -137,11 +146,38 @@ The options below are all specified on the command-line.
   capture, it is possible to use [`discard_check_output`](#discard_check_output).
 
 - `-client` ((#\_client)) - The address to which Consul will bind client
-  interfaces, including the HTTP and DNS servers. By default, this is "127.0.0.1",
+  interfaces, including the HTTP, HTTPS, gRPC and DNS servers. By default, this is "127.0.0.1",
   allowing only loopback connections. In Consul 1.0 and later this can be set to
   a space-separated list of addresses to bind to, or a [go-sockaddr]
   template that can potentially resolve to multiple addresses.
 
+  <CodeBlockConfig hideClipboard>
+  
+  ```shell
+  # Bind consul client interfaces to private IPv4 interfaces.
+  $ consul agent -dev -client '{{ GetPrivateInterfaces | exclude "type" "ipv6" | join "address" " " }}'
```

Comment:
```suggestion
$ consul agent -dev -client '{{ GetPrivateInterfaces | exclude "type" "ipv6" | join "address" " " }}'
```

Reply:
Remove the comment as it will be included with the clip board

Reply:
If we're moving all of the comments into headings, should we also expose the clipboard since there won't be comments to worry about in a copy?

I looked like in other spots of the documentation we were hiding the clipboard when it was a shell command.